stories/dynamic-data.stories.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { storiesOf } from '@storybook/react'
+
+import BarChart from '../src/components/bar-chart/'
+import PieChart from '../src/components/pie-chart'
+import barChartData from './data/bar-chart-data'
+import pieChartData from './data/pie-chart-data'
+import ResponsiveChartWrapper from './responsive-chart-wrapper'
+
+
+const title = 'My Title'
+
+storiesOf('Dynamic Data', module)
+  .add('Dynamic Data - legend update', () => {
+    const [data, setData] = useState(barChartData)
+    return (
+      <ResponsiveChartWrapper>
+        <button
+          onClick={ () => setData([{
+            address_city: 'Sauga',
+            dopeness: 1000,
+            awesomeness: 800,
+          },
+          {
+            address_city: 'T-Dot',
+            dopeness: 2000,
+            awesomeness: 2100,
+          }]) }
+        >Change Data</button>
+        <br />
+        <BarChart
+          data={ data }
+          axisBottomLegendLabel='Address City'
+          axisLeftLegendLabel='Visitors'
+        />
+      </ResponsiveChartWrapper>
+    )
+  })
+  .add('Dynamic Data - tooltip update', () => {
+    const [data, setData] = useState(pieChartData)
+    return (
+      <ResponsiveChartWrapper>
+        <button
+          onClick={ () => setData([
+            { address_city: 'Sauga', awesomeness: 800 },
+            { address_city: 'T-Dot', awesomeness: 1000 },
+            { address_city: 'Markham', awesomeness: 1500 }
+          ]) }
+        >Change Data</button>
+        <PieChart title={ title } data={ data } isDonut={ false } />
```

Comment:
based on a quick research, the error we're experiencing here doesn't seem related to tooltip. I guess it might be something in the `ResponsiveChartWrapper` measurements. I'm saying that bc [`getBoundingClientRect`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) is related to sizes and position and the error says that the issue is in the children created by Measure from ResponsiveWrapper. Might need a deeper look/investigation with a fresh mind...
![image](https://user-images.githubusercontent.com/53827690/94564625-2fe5a200-0236-11eb-8c29-f319bedcb671.png)

Reply:
I think this is fixed in https://github.com/EQWorks/chart-system/pull/122